internal/host/transition.go

```diff
@@ -363,6 +355,20 @@ func (th *transitionHandler) PostPrepareForInstallation(sw stateswitch.StateSwit
 		statusInfoPreparingForInstallation)
 }
 
+func (th *transitionHandler) resetHostFields(ctx context.Context, log logrus.FieldLogger, db *gorm.DB, state *stateHost,
```

Comment:
Not sure if this function is needed, maybe you can just a have a list of field to reset and use it as another parameter to `updateTransitionHost` and `updateHostProgress` ?

Reply:
I don't think it is possible to create a global array of interfaces (e.g. `bootstrap` is boolean and `inventory` is string)